simulator/src/test/java/com/hedera/block/simulator/BlockStreamSimulatorTest.java

```diff
@@ -92,10 +92,24 @@ void start_exitByBlockNull() throws InterruptedException, ParseException, IOExce
         assertTrue(blockStreamSimulator.isRunning());
     }
 
+    void start_usingConfigurationConstructor()
+            throws InterruptedException, ParseException, IOException {
+        blockStreamSimulator = new BlockStreamSimulatorApp(configuration);
+        blockStreamSimulator.start();
+        assertTrue(blockStreamSimulator.isRunning());
+    }
+
     private String getAbsoluteFolder(String relativePath) {
         return Paths.get(relativePath).toAbsolutePath().toString();
     }
 
+    @Test
+    void start_usingEmptyConstructor() throws IOException, InterruptedException, ParseException {
+        blockStreamSimulator = new BlockStreamSimulatorApp();
+        blockStreamSimulator.start();
+        assertTrue(blockStreamSimulator.isRunning());
```

Comment:
Maybe we can expand these tests using mocks, etc to check that get default configuration is actually present and correct, test the `stop()` method...